client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -43,6 +46,60 @@ const CancelPurchaseRefundInformation = ( {
 		} );
 	};
 
+	const featuresList = ( productSlug, hasDomain ) => {
+		if ( typeof productSlug !== 'string' ) {
+			return null;
+		}
+
+		const planCancellationFeatures = getPlanCancellationFeatures( productSlug, hasDomain );
+
+		return (
+			<Fragment>
+				<ul className="cancel-purchase__refund-information--list-plan-features">
+					{ planCancellationFeatures.featureList.map( ( cancellationFeature ) => {
+						return (
+							<li key={ cancellationFeature }>
+								<Gridicon
+									className="cancel-purchase__refund-information--item-cross-small"
+									size={ 24 }
+									icon="cross-small"
+								/>
+								{ getCancellationFeatureByKey( cancellationFeature ) }
+							</li>
+						);
+					} ) }
+					{ planCancellationFeatures.andMore && (
+						<li
+							className="cancel-purchase__refund-information--item-more"
+							key="cancellationAndMore"
+						>
+							<span className="cancel-purchase__refund-information--item-more-span">
+								{ i18n.translate( 'And more' ) }
+							</span>
+						</li>
+					) }
+				</ul>
+			</Fragment>
+		);
+	};
+
+	const cancelSubscriptionDescription = () => {
+		const planSlug = site.plan.product_slug;
+
+		return (
+			<div>
+				<p className="cancel-purchase__refund-information-subtitle">
+					{ i18n.translate(
+						'If you cancel your plan subscription your site may appear broken and things may not work properly.'
```

Comment:
@dzver  @southp  we don't have a valid fallback for this string and it's the copy header for the feature set
<img width="728" alt="Screenshot 2022-12-09 at 16 58 41" src="https://user-images.githubusercontent.com/5706607/206742222-0916ef81-0ea9-4d36-a337-73be9a682bae.png">

Reply:
Let's add String Freeze for now, since we have to wait for other things anyway.

Reply:
string freeze was added